components/add_groups_to_channel_modal/add_groups_to_channel_modal.test.jsx

```diff
@@ -59,7 +61,7 @@ describe('components/AddGroupsToChannelModal', () => {
         expect(wrapper.state('addError')).toEqual(message);
     });
 
-    /*test('should match state when handleSubmit is called', async () => {
+    test('should match state when handleSubmit is called', async () => {
         const linkGroupSyncable = jest.fn().
             mockResolvedValueOnce({error: true}).
             mockResolvedValue({data: true});
```

Comment:
Here, the initial author of the test didn't correctly make the mock of `linkGroupSyncable` function which was causing the function to fail below. The following change should make it so that `handleHide` is not called.
```suggestion
        const linkGroupSyncable = jest.fn().mockResolvedValue({error: true, data: true});
```
The way it's currently written, only on the first call of the function will we get a `error = true` and afterwards error will just be `undefined` which is why the `handleHide` was being called once because of the second function (There are 2 ids that are looped through) call happening. I persume the author meant for error to return true every time hence why they said `handleHide.not.called`.